libs/ui/src/diagnostics/mark_scan_paper_handler_section.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { assert } from '@votingworks/basics';
+import { DiagnosticRecord } from '@votingworks/types';
+import { H2, P } from '../typography';
+import { InfoIcon, SuccessIcon, WarningIcon } from './icons';
+import { Button } from '../button';
+
+export interface MarkScanPaperHandlerSectionProps {
+  mostRecentPaperHandlerDiagnostic?: DiagnosticRecord;
+  isPaperHandlerDetected: boolean;
+  onStartPaperHandlerDiagnostic: () => void;
+}
+
+export function MarkScanPaperHandlerSection({
+  mostRecentPaperHandlerDiagnostic,
+  isPaperHandlerDetected,
+  onStartPaperHandlerDiagnostic,
+}: MarkScanPaperHandlerSectionProps): JSX.Element {
+  if (mostRecentPaperHandlerDiagnostic) {
+    assert(mostRecentPaperHandlerDiagnostic.type === 'mark-scan-paper-handler');
+  }
+
+  return (
```

Comment:
I'll probably create a common component in the PR that adds the PAT diagnostic based on what similarities emerge between all the diagnostics.

Reply:
https://github.com/votingworks/vxsuite/pull/4784